oar/cli/cmd_stage_testing.py

```diff
@@ -36,12 +35,20 @@ def stage_testing(ctx, build_number):
                         logger.exception("trigger stage pipeline job failed")
                         raise
                     logger.info(f"triggerred stage pipeline job: <{build_url}>")
-                    nm.sc.post_message(cs.get_slack_channel_from_contact("qe"), "["+cs.release+"] stage testing job: "+build_url)
+                    nm.sc.post_message(cs.get_slack_channel_from_contact("qe-release"), "["+cs.release+"] stage testing job: "+build_url)
                     report.update_task_status(LABEL_TASK_STAGE_TEST, TASK_STATUS_INPROGRESS)
                 else:
                     logger.info("push to cdn stage is not completed, so will not trigger stage test")
         except Exception as we:
             logger.exception("trigger stage testing failed")
     else: 
         logger.info(f"check stage job status according to job id:{build_number}")
-        jh.get_job_status(jh.zstream_url, "Stage-Pipeline", build_number)
+        job_status= jh.get_job_status(jh.zstream_url, "Stage-Pipeline", build_number)
+        logger.info(f"{job_status}")
```

Comment:
```suggestion
        logger.info(job_status)
```

Reply:
@rioliu-rh  I deleted this, it's debug info